R/help.R

```diff
@@ -323,11 +336,22 @@ arg_descriptions_from_doc_default <- function(args, doc) {
 
 # Extract arguments descriptions for docs in Sphinx style
 arg_descriptions_from_doc_sphinx <- function(doc) {
+  
   doctree <- sphinx_doctree_from_doc(doc)
   params <- doctree$ids$parameters$children[[2]]$children
-  sapply(params, function(param) {
-    param$children[[3]]$astext()
+  
+  output <- lapply(params, function(param) {
+    children <- param$children
+    children[[length(children)]]$astext()
```

Comment:
I was seeing that in some cases we had two children in this node rather than three. I'm guessing the last entry is always the description, so we just grab that rather than hardcoding the number.